spec/xcpretty/parser_spec.rb

```diff
@@ -225,6 +225,17 @@ module XCPretty
       @parser.parse(SAMPLE_TOUCH)
     end
 
+    it "parses write file" do
+      @formatter.should receive(:format_write_file).with(
+      '/Users/me/myproject/Build/Intermediates/Pods.build/Debug-iphonesimulator/Pods-AFNetworking.build/Objects-normal/x86_64/Pods-AFNetworking.LinkFileList')
```

Comment:
Line is too long. [158/80]<br>Indent the first parameter one step more than the previous line.